02-data-types-and-variables/README.md

```diff
@@ -207,3 +207,141 @@ Select debug tab of Visual Studio Code and click `Start Debugging` or click `F5`
 
 </p>
 </details>
+
+## Exercise 2.5: let's make it look more like animal shelter page
+
+Update home page text for it to be more "animal shelter like".
+
+<details>
+<summary>Solution</summary>
+<p>
+
+You need to update `Index.razor` file in folder `Pages`. This is the file that represents our home page. Updated page file should look like this:
+
+```
+@page "/"
+
+<h1>Welcome to our animal shelter page!</h1>
+
+Here you can adopt you future pet.
+
+```
+
+</p>
+</details>
+
+## Exercise 2.6: make animal registration page
+
+Use `Counter` page as a base for the task as it is very similar to the initial version of `Animal Registration` page.
+
+Rename menu item `Counter` to `Animal Registration`.
+
+Update page to count animals, rename `Click me` to `Add animal`, add button `Remove animal` and implemet the logic behind it. 
+Please make sure that methods, variables and files themselves are named properly and their names represent their purpose.
+
+Instead of `Current count:` display `Currently we have *animal number goes here* animals`
+
+<details>
+<summary>Solution</summary>
+<p>
+
+### Step 1
+Rename `Counter` to `Animal Registration` in `NavMenu.razor` file under `Shared` folder. This file represents our menu. After update your updated line should look like this:
+
+```
+<span class="oi oi-plus" aria-hidden="true"></span> Animal Registration
+
+```
+
+### Step 2
+Rename file `Counter.razor` to `AnimalRegistration.razor`.
+Change first line of the file to `@page "/animalregistration"`.
+Change menu item in `NavMenu.razor` to point to the new page name which makes more sense.
+```
+<NavLink class="nav-link" href="animalregistration">
+    <span class="oi oi-plus" aria-hidden="true"></span> Animal Registration
+</NavLink>
+```
+We do this so our names make more sense and it's easy to understand file purpose from it's name.
```

Comment:
We use purposeful names for files and code so that other developers who will read your code would have an easier time understanding it. Also, for yourself when you return to the code after some time.